src/animations/mandelbrot.rs

```diff
@@ -0,0 +1,87 @@
+use super::Animation;
+use crate::frame::{value_to_char, Character, Color, Frame, HSVColor};
+use nalgebra::{Complex, ComplexField};
+
+pub struct Mandelbrot {
+    width: f64,
+}
+
+impl Default for Mandelbrot {
+    fn default() -> Mandelbrot {
+        Mandelbrot { width: 8.0 }
+    }
+}
+
+impl Mandelbrot {
+    const NAME: &'static str = "Mandelbrot";
+    const AUTHOR: &'static str = "Marco";
+    const BOUND: f64 = 2.0;
+
+    fn eval(&mut self, c: Complex<f64>) -> i32 {
+        let mut z = Complex::new(0.0, 0.0);
+
+        let max_iterations = self.calculate_max_iterations();
+
+        for iteration in 0..max_iterations {
+            z = z.powf(2.0) + c;
```

Comment:
```suggestion
            z = z.powi(2) + c;
```

Using powi might be faster in this case.